content/develop/_index.md

```diff
@@ -10,35 +10,89 @@ tags = [ "theme" ]
     - If it is a `container` component, then it has surface colors and padding, no typography styles!
     - Read more on this below.
 3. Context aware
     - `Content` components should be equipped with styles for the `on="dark"` attribute, so that they can be used on both light and dark backgrounds. 
 4. Framework agnostic
-    - A RHElement should “just work” when you drop it onto any page (provided the proper polyfills are there). It should have ALL the styles it needs, such as font-family properties.
+    - A PatternFly Element should "just work" when you drop it onto any page (provided the proper polyfills are there). It should have ALL the styles it needs, such as font-family properties.
 
-## Container RHElements
+## Container Elements
 
-A container never imposes padding or styles on the children. It is only concerned with their horizontal or vertical alignment and spacing. For example in a card, you might apply margins to the slots, but you would not apply padding or color not to the content inside of slots.
+A container never directly styles the children inside the contain. It is only concerned with their horizontal or vertical alignment and spacing. For example in a card, you might apply margins to the slots, but you would not apply padding or color not to the content inside of slots.
 
-**Example: [rh-card](https://github.com/RHElements/rhelements/blob/master/elements/rh-card/src/rh-card.scss)**
+**Example: [Card](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-card/src/pfe-card)**
 
 This component has background color options, and attributes to change the vertical spacing between the slots.
 
-## Content RHElements
+## Content Elements
 
 A component always touches all four sides of its parent container. For example, if you are creating a teaser component, it should not have padding or margins around the type. It might, however have margins between the headline and author.
 
-**Example: [rh-cta](https://github.com/RHElements/rhelements/blob/master/elements/rh-cta/src/rh-cta.scss)**
+**Example: [Call-to-action](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-cta/src/pfe-cta)**
 
-This component has typography styles, and attributes to change the priority level of the call-to-action. It also has `on="dark"` styles built in, so that colors will automatically adjust. 
\ No newline at end of file
+This component has typography styles, and attributes to change the priority level of the call-to-action. It also has `on="dark"` styles built in, so that colors will automatically adjust. 
+
+## Combo Elements
+
+These elements are designed to create easy-to-use combinations of other smaller components. Ideally they also bundle up attributes into a few configurations.
+
+**Example: [Hide-show](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-hide-show)**
+
+
+## @todo should features be opt-in or opt-out?
```

Comment:
Hmm... here's an example: Let's say we decide the pagination should include previous and next links by default, but the jump-link is a bonus feature.

If the attributes are booleans, that would mean that this markup:
`<pfe-pagination jump-link prev next>`
would turn on the jump-link, and turn off previous and next buttons (which is confusing in and of itself). 

Later, if we decide the jump-link should come standard, then that same markup would turn off the jump-link, which is less than desirable.

Reply:
There's a nuance here though that I think makes it simpler:

`<pfe-pagination include-jump-link include-prev-next="false">`

The verbosity helps a lot but also the use of false for booleans makes things clearer.  So if the design system updates and you're using `<pfe-pagination>` you just get whatever is the default.  If you've got `<pfe-pagination include-jump-link>` you'll still get the jump link whether it's the default or not.  Now if you're using the attribute without and the default changes and you _don't_ want the change, you'd have to update to `<pfe-pagination include-jump-link="false">` to specifically turn it off.

Reply:
I like this approach. One more question though :)

Is  `include-jump-link="true"` == `include-jump-link` ? In concept it is, but in code it is not. The only reason I think this (could) matter is because if I am editing an existing page, which has `<pfe-pagination include-jump-link="false">` and I decide to turn on the jump link, it would be easy for me to assume that `<pfe-pagination include-jump-link="true">` should turn on that feature, but that syntax would not be supported.  

If its not something we are going to support in code, we should document the heck out of it :)

Reply:
JK, even in CSS its not a problem: https://codepen.io/starryeyez024/pen/QzKeVv